src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -31,7 +31,7 @@
   private static String[] testClassPaths;
 
   /** The additional class loaders to find classes in. */
-  private static final List<ClassLoader> classLoaders = new Vector<>();
+  private static final List<ClassLoader> classLoaders = Lists.newArrayList();
```

Comment:
Be careful, Vectors are synchronized. I don't know if we need it or not (I supposed we don't).